content/building-blocks/automate-and-execute-your-work/reproducible-work/google_cloud_docker.md

```diff
@@ -0,0 +1,312 @@
+---
+title: "Import and Run a python environment on Google cloud with Docker" 
+description: "Learn how to import and run a python environment on google cloud with docker"

```

Comment:
The ideal lenght of the description is 145 characters. This description is 76. An example of a longer description would be: Learn to install and set up Docker in a Google Cloud instance, build environments, and connect docker with Google Cloud storage buckets…

Reply:
Perfect! I've modified it to the 150 character long: "Learn how to import a containerized python environment with docker, run it on a google cloud instance and connect it with Google cloud storage buckets"